service/src/main/java/org/apache/griffin/core/util/FSUtil.java

```diff
@@ -177,4 +177,23 @@ private static void checkHDFSConf() {
         }
     }
 
+    public static String getFirstMissRecordPath(String hdfsDir) throws Exception{
+        List<FileStatus> fileList = listFileStatus(hdfsDir);
+        for(int i=0; i<fileList.size();i++){
+            if(fileList.get(i).getPath().toUri().toString().toLowerCase().contains("missrecord")){
+                return fileList.get(i).getPath().toUri().toString();
+            }
+        }
+        return null;
```

Comment:
Is it okay to return `null`? Because it seems it can be used in `isFileExists` method, and there is no null checking.
Please let me know if I'm thinking wrong. Thanks.